File: packages/fn-cli/templates/handler/python/handler.py
"""____name____ — handler implementation."""
from typing import Any


async def handler(params: dict[str, Any]) -> dict[str, Any]:
    """Process a job payload and return a JSON-serialisable result."""
    # TODO: implement
    print(f"____name____ invoked: {params}")
    return {"ok": True}

File: packages/fn-cli/templates/handler/node-graphql/handler.ts
import type { FunctionHandler } from '@constructive-io/fn-runtime';

interface Payload {
  // TODO: shape your function input
}

interface Result {
  ok: true;
}

const handler: FunctionHandler<Payload, Result> = async (params, ctx) => {
  ctx.log.info('____name____ invoked', params);
  return { ok: true };
};

export default handler;
